scls-cddl/cddl-src/Cardano/SCLS/Namespace/Pots/V0.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Use camelCase" #-}
+
+module Cardano.SCLS.Namespace.Pots.V0 where
+
+import Cardano.SCLS.Common
+import Codec.CBOR.Cuddle.Comments ((//-))
+import Codec.CBOR.Cuddle.Huddle
+import Data.Function (($))
+import Text.Heredoc (str)
+
+-- | Top-level entry for pots namespace
+record_entry :: Rule
+record_entry = "record_entry" =:= pots_table
+
+-- | Table mapping pot names to coin values
+pots_table :: Rule
+pots_table =
+  comment
+    [str|Pots table containing the various accounting pots in Cardano
+        |
+        |Fields:
+        |  - Treasury: coins in the treasury
+        |  - Reserves: coins in the reserves
+        |]
+    $ "pots_table"
+      =:= mp
+        [ "treasury" ==> coin
+        , "reserves" ==> coin
```

Comment:
I don't quite follow the argument that we don't have all of these pots yet. They are defined in various places throughout the NewEpochState, my proposal was to reformat the state slightly and pull them all together.

Reply:
Hm.. got you! I though that it was just about ChainAccountState (previously AccountState).